### PR TITLE
DMU 6 Cards + 2 tokens 

### PR DESCRIPTION
**Pull request by Simisays**
6 Read ahead Sagas from DMU + 2 new tokens

also is there a way to contribute token images of new tokens (correctly named) to the file server ?

### CODE REVIEW

File: forge-gui/res/tokenscripts/c_0_2_ornithopter_flying.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Name:Ornithopter
```
Comment (paulsnoops) [inline forge-gui/res/tokenscripts/c_0_2_ornithopter_flying.txt:1]: The file for this one just needs to be named ornithopter.txt as it's a "named" token and this is what the edition file shows

File: forge-gui/res/cardsfolder/upcoming/braids_frightful_return.txt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Name:Braids's Frightful Return
+ManaCost:2 B
+Types:Enchantment Saga
+K:Read ahead:3:DBSacrifice,DBChangeZone,DBSacrificeOpp
+SVar:DBSacrifice:DB$ Sacrifice| Optional$ True | Defined$ You | RememberSacrificed$ True | SacValid$ Creature | Amount$ 1 | SubAbility$ DBDiscard | SpellDescription$ You may sacrifice a creature. If you do, each opponent discards a card.
+SVar:DBDiscard:DB$ Discard | Mode$ TgtChoose | Defined$ Player.Opponent | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ GE1 | NumCards$ 1 | ConditionPresent$ Card | SubAbility DBCleanup
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ValidTgts$ Creature.YouOwn | SpellDescription$ Return target creature card from your graveyard to your hand.
+SVar:DBSacrificeOpp:DB$ Sacrifice | ValidTgts$ Player | SacValid$ Permanent.nonLand+nonToken | RememberSacrificed$ True | SubAbility$ DBDraw | SpellDescription$ Target opponent may sacrifice a nonland, nontoken permanent. If they don't, they lose 2 life and you draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/braids_frightful_return.txt:8]: `ValidTgts$ Opponent`

File: forge-gui/res/cardsfolder/upcoming/founding_the_third_path.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Founding the Third Path
+ManaCost:1 U
+Types:Enchantment Saga
+K:Read ahead:3:DBPlayFree,DBMill,DBCopy
+SVar:DBPlayFree:DB$ Play | Valid$ Instant.YouOwn,Sorcery.YouOwn | ValidSA$ Spell.cmcEQ1,Spell.cmcEQ2 | ValidZone$ Hand | WithoutManaCost$ True | Amount$ 1 | Controller$ You | Optional$ True | SpellDescription$ You may cast an instant or sorcery spell with mana value 1 or 2 from your hand without paying its mana cost.
+SVar:DBMill:DB$ Mill | NumCards$ 4 | ValidTgts$ Player | SpellDescription$ Target player mills four cards.
+SVar:DBCopy:DB$ ChangeZone | TargetMin$ 0 | TargetMax$ 1 | Origin$ Graveyard | Destination$ Exile | TgtPrompt$ Choose up to one target instant or sorcery card in your graveyard | ValidTgts$ Instant.YouCtrl,Sorcery.YouCtrl | RememberChanged$ True | SubAbility$ DBPlay | SpellDescription$ Exile target instant or sorcery card from your graveyard. Copy it. You may cast the copy.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/founding_the_third_path.txt:7]: Why TargetMin?

File: forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Name:The Phasing of Zhalfir
+ManaCost:2 U U
+Types:Enchantment Saga
+K:Read ahead:3:DBPhase,DBPhase,DBDestroyAll
+SVar:DBPhase:DB$ Phases | ValidTgts$ Permanent.nonLand+Other | WontPhaseInNormal$ True | ConditionPresent$ Card.Self | SubAbility$ DBEffect
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt:5]: Missing SpellDescription

File: forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+SVar:DBExileSelf:DB$ ChangeZone | Origin$ Command | Destination$ Exile | Defined$ Self
+SVar:DBDestroyAll:DB$ DestroyAll | ValidCards$ Creature | RememberDestroyed$ True | SubAbility$ DBRepeat | SpellDescription$ Destroy all creatures. For each creature destroyed this way, its controller creates a 2/2 black Phyrexian creature token.
+SVar:DBRepeat:DB$ RepeatEach | UseImprinted$ True | DefinedCards$ DirectRemembered | RepeatSubAbility$ DBToken | SubAbility$ DBCleanup | ChangeZoneTable$ True
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenOwner$ ImprintedController | TokenScript$ b_2_2_phyrexian
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ true
+DeckHas:Ability$Tokens
+Oracle:Read ahead (Choose a chapter and start with that many lore counters. Add one after your draw step. Skipped chapters don't trigger. Sacrifice after III.)\nI — You may sacrifice a creature. If you do, each opponent discards a card.\nII — Return target creature card from your graveyard to your hand.III — Target opponent may sacrifice a nonland, nontoken permanent. If they don’t, they lose 2 life and you draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt:15]: Wrong Oracle

File: forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt [hunk 6]
```diff
@@ -0,0 +1,15 @@
+Name:The Phasing of Zhalfir
+ManaCost:2 U U
+Types:Enchantment Saga
+K:Read ahead:3:DBPhase,DBPhase,DBDestroyAll
+SVar:DBPhase:DB$ Phases | ValidTgts$ Permanent.nonLand+Other | WontPhaseInNormal$ True | ConditionPresent$ Card.Self | SubAbility$ DBEffect
+SVar:DBEffect:DB$ Effect | Triggers$ TrigComeBack | RememberObjects$ Targeted | ImprintCards$ Self | ConditionPresent$ Card.Self | Duration$ Permanent | ForgetOnPhasedIn$ True
+SVar:TrigComeBack:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Card.IsImprinted | Execute$ DBPhaseIn | TriggerZones$ Command | TriggerController$ TriggeredCardController | Static$ True | TriggerDescription$ Target creature phases out until CARDNAME leaves the battlefield. Tap that creature as it phases in this way.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt:7]: Remove TriggerDescription

File: forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+SVar:DBPhaseIn:DB$ Phases | Defined$ Remembered | Tapped$ True | PhaseInOrOut$ True | SubAbility$ DBExileSelf
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt:8]: Not Tapped...

File: forge-gui/res/cardsfolder/upcoming/braids_frightful_return.txt [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+SVar:DBDraw:DB$ Draw | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ EQ0 | SubAbility$ DBLoseLife
+SVar:DBLoseLife:DB$ LoseLife | ConditionDefined$ Remembered | ConditionPresent$ Card | ConditionCompare$ EQ0 | LifeAmount$ 2 | Defined$ Targeted | SubAbility DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckHas:Ability$Discard|Sacrifice
+Oracle:Read ahead (Choose a chapter and start with that many lore counters. Add one after your draw step. Skipped chapters don't trigger. Sacrifice after III.)\nI — You may sacrifice a creature. If you do, each opponent discards a card.\nII — Return target creature card from your graveyard to your hand.III — Target opponent may sacrifice a nonland, nontoken permanent. If they don't, they lose 2 life and you draw a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/braids_frightful_return.txt:13]: Oracle missing linebreak

File: forge-gui/res/cardsfolder/upcoming/yotia_declares_war.txt [hunk 9]
```diff
@@ -0,0 +1,10 @@
+Name:Yotia Declares War
+ManaCost:1 R
+Types:Enchantment Saga
+K:Read ahead:3:DBToken,DBTap,DBAnimate
+SVar:DBToken:DB$ Token | TokenScript$ ornithopter | SpellDescription$ Create a 0/2 colorless Thopter artifact creature token with flying named Ornithopter.
+SVar:DBTap:AB$ DealDamage | Cost$ tapXType<X/Artifact> | CostDesc$ As an additional cost to cast this spell, tap any number of untapped artifacts you control. | ValidTgts$ Creature,Planeswalker | NumDmg$ X | SpellDescription$ Tap any number of untapped artifacts you control. When you do, NICKNAME Declares War deals that much damage to target creature or planeswalker.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/yotia_declares_war.txt:6]: this is reflexive trigger

File: forge-gui/res/tokenscripts/b_2_2_phyrexian.txt [hunk 10]
```diff
@@ -0,0 +1,6 @@
+Name:Phyrexian
```
Comment (tool4ever) [inline forge-gui/res/tokenscripts/b_2_2_phyrexian.txt:1]: ` Name:Phyrexian Token`

File: forge-gui/res/cardsfolder/upcoming/yotia_declares_war.txt (hunk 9 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/yotia_declares_war.txt]: >You don't choose the target creature or planeswalker as you put the second chapter ability of Yotia Declares War on the stack. First, you tap any number of artifacts. Then, a second "reflexive" ability triggers (denoted by the phrase "when you do") that requires a target. Players may respond to the reflexive triggered ability as normal, and they will know how many artifacts were tapped when they do.

Check out CR 603.12 and the example there for more details

File: forge-gui/res/cardsfolder/upcoming/yotia_declares_war.txt [hunk 11]
```diff
@@ -0,0 +1,11 @@
+Name:Yotia Declares War
+ManaCost:1 R
+Types:Enchantment Saga
+K:Read ahead:3:DBToken,TrigTap,DBAnimate
+SVar:DBToken:DB$ Token | TokenScript$ ornithopter | SpellDescription$ Create a 0/2 colorless Thopter artifact creature token with flying named Ornithopter.
+SVar:TrigTap:AB$ ImmediateTrigger | Cost$ tapXType<X/Artifact> | Execute$ DBDamage | SpellDescription$ Tap any number of untapped artifacts you control. When you do, CARDNAME deals that much damage to target creature or planeswalker.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/yotia_declares_war.txt:6]: `Cost$ Mandatory tapXType<X/Artifact>`

File: forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+SVar:DBPhaseIn:DB$ Phases | Defined$ Remembered | PhaseInOrOut$ True | SubAbility$ DBExileSelf
+SVar:DBExileSelf:DB$ ChangeZone | Origin$ Command | Destination$ Exile | Defined$ Self
+SVar:DBDestroyAll:DB$ DestroyAll | ValidCards$ Creature | RememberDestroyed$ True | SubAbility$ DBRepeat | SpellDescription$ Destroy all creatures. For each creature destroyed this way, its controller creates a 2/2 black Phyrexian creature token.
+SVar:DBRepeat:DB$ RepeatEach | UseImprinted$ True | DefinedCards$ DirectRemembered | RepeatSubAbility$ DBToken | SubAbility$ DBCleanup | ChangeZoneTable$ True
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenOwner$ ImprintedController | TokenScript$ b_2_2_phyrexian
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ true
+DeckHas:Ability$Tokens
+Oracle:Read ahead (Choose a chapter and start with that many lore counters. Add one after your draw step. Skipped chapters don’t trigger. Sacrifice after III.)\nI, II — Another target nonland permanent phases out. It can’t phase in for as long as you control The Phasing of Zhalfir.\nIII — Destroy all creatures. For each creature destroyed this way, its controller creates a 2/2 black Phyrexian creature token.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/the_phasing_of_zhalfir.txt:15]: don’t = don't
(curly)

**Comment by paulsnoops** [comment]
To asnwer your question: @kevlahnota provides the card and token images on the ftp

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**